packages/suite/src/utils/wallet/reviewTransactionUtils.ts

```diff
@@ -21,6 +21,12 @@ export const getIsUpdatedSendFlow = (device: TrezorDevice) => {
     return versionUtils.isNewerOrEqual(firmwareVersion, '2.6.0');
 };
 
+export const getIsUpdatedRbfFlow = (device: TrezorDevice) => {
+    const firmwareVersion = getFirmwareVersion(device);
+
+    return versionUtils.isNewerOrEqual(firmwareVersion, '2.6.3');
+};
+
```

Comment:
There is no need for this new condition, because it's not a new change, but old bug.

Reply:
0d367da89c74a32712162743a3b9b0b3cc8944a1